### ISSUE TITLE
关于config文件里的n_cats设置

### ISSUE THREAD

**Issue by yangaiping** [CoinCheung/BiSeNet#314]
我通过check_dataset_info.py脚本查到我的所有数据集标签图像像素值区间是[0，132]，但是我的数据集标签类别一共才13类，为什么n_cats设成13我的训练就会报错呢，但是设成133又能成功运行？
![image](https://github.com/CoinCheung/BiSeNet/assets/109123962/3b104359-1120-412b-93bb-b43c6fccf7fb)

**Comment by CoinCheung** [comment]
Your category does not in range of [0, 13], but in [0, 131] instead.  You should convert the label values into [0, 13].

**Comment by yangaiping** [comment]
您是什么意思呢，我的图像像素值区间是[0，132]，但是我的像素颜色种类是13，那我的n_cats应该设为多少呢

**Comment by CoinCheung** [comment]
You should map the pixel value of  your **label file** into range of [0, 12].

**Comment by yangaiping** [comment]
代码里的这个输出[0，132]难道不是像素的最小值与最大值形成的区间吗
![image](https://github.com/CoinCheung/BiSeNet/assets/109123962/0ae31892-d8c9-4f12-a124-26310f82950c)
![image](https://github.com/CoinCheung/BiSeNet/assets/109123962/126fb344-f08f-4752-acfc-c74723f41f3b)

**Comment by CoinCheung** [comment]
You have 13 categories, but your category index are not continously ranging from 0 to 12, they are sparsely ranging from 0 to 132.

You should map your raw label values continuously into range [0, 12].

For example, in the picture above, we can see the mapping rule as:
```
0    -> 0
14  -> 1
19  -> 2
....
```

**Comment by yangaiping** [comment]
您这里面放置的标签文件是彩色标签图片还是单通道8位标签图像呢
![image](https://github.com/CoinCheung/BiSeNet/assets/109123962/648b3bdb-c11b-4bd2-9ff7-ddf334422c2f)

**Comment by CoinCheung** [comment]
They are expected to be singe channel image.

**Comment by yangaiping** [comment]
我明白您的意思了，我的png格式的标签文件有问题，我应该下去修改，是需要把彩色标签图像改成单通道8位标签图像吗

**Comment by CoinCheung** [comment]
Yes, they are expected to be single channel with label values rather than pixel values.

**Comment by yangaiping** [comment]
非常感谢您，我再下去试试。再次感谢

**Comment by inlaid** [comment]
would you please explain the detailed meanings of the parameters in the configuration file, such as bisenet_customer.py, including parameters like num_aux_heads and eval_scales, and so on. Thanks!
